CONTRIBUTING.md

```diff
@@ -74,6 +74,40 @@ include a Signed-off-by line. You can find more information about this [here](ht
 For more advice on how to contribute, follow that link:
 https://osrd.fr/en/docs/guides/contribute/contribute-code
 
+### Additional targets
+
+Some additional targets are currently officially supported. \
```

Comment:
Can we avoid the blackslahes?

Standard Markdown creates paragraphs unless a full blank line is used as separator.